frontend/app/model/Eventbrite.scala

```diff
@@ -46,24 +46,28 @@ object Eventbrite {
     lazy val blurb = truncateToWordBoundary(text, 120)
   }
 
-  case class EBAddress(address_1: Option[String],
-                       address_2: Option[String],
-                       city: Option[String],
-                       region: Option[String],
-                       postal_code: Option[String],
-                       country: Option[String]) extends EBObject {
-
+  case class EBAddress(
```

Comment:
It feels a bit weird that all these fields are options. Is there a good reason for that? You wouldn't need something like `seqToLine` if they were just strings